client/scss/layouts/_login.scss

```diff
@@ -23,32 +33,60 @@ $desktop-nice-padding: 100px;
   }
 
   h1 {
-    font-weight: 300;
-    font-size: 2em;
-    line-height: 1em;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
     white-space: nowrap;
+    font-weight: 800;
+    font-size: 30px;
+    line-height: 130%;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex: none;
+    flex-grow: 0;
+    z-index: 1;
+    margin: 0px;
   }
 
   form {
-    width: 100%;
-
-    ul {
-      @include unlist();
-    }
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 50px 60px 40px;
+    gap: 30px;
+    isolation: isolate;
```

Comment:
Be sure you need this. It's probably good to try to get your design working without any z-index / isolate where possible. Will make life simpler